### ISSUE TITLE
Break repo into individual designs

### ISSUE THREAD

**Issue by jonnew** [open-ephys/commutators#14]
The repo, as of writing this issue, needs to be divided into 4 repos. The first is the control board and firmware

1. commutator-controller
    - Control board schematic, PCB, assembly materials
    - Firmware

The additional three are complete designs with (1) included as submodule:

1. commutator-spi
3. commutator-coax
4. commutator-coax-dual

**Comment by cjsha** [comment]
Sure. We decided this will be the last step in Revision D milestone. Two questions:
- Should we preserve git history of each of the new four repos? https://docs.github.com/en/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository Or should each new repo get a fresh git history?
- Should we archive this repo? Iirc, once archived, you can't rename an archived repo so we should consider renaming and then archiving.

Depending on which fiber-coax rotary joint we go with, I suggest calling it "commutator-dual" instead of "commutator-coax-dual" if it accommodates both the fiber-coax rotary joint and coax-coax rotary joint.

**Comment by cjsha** [comment]
https://github.com/open-ephys/commutator-dual
https://github.com/open-ephys/commutator-spi
https://github.com/open-ephys/commutator-coax
https://github.com/open-ephys/commutator-controller

They are still private github repos - I just wanna double-check that the licensing, READMEs, how organized teensy designs vs rp2040 designs (i.e. rev c & before vs rev d & after), etc. are ok.

**Comment by jonnew** [comment]
Looks good. I can review when I get in, then we can deprecate this one and make public. 

We need to update the docs links too, right?

**Comment by cjsha** [comment]
Yeah, I made an [issue](https://github.com/open-ephys/commutator-docs/issues/17) just now. Imo, we don't have to immediately phase out the old repo's links from the docs. If we archive this repo, the links will still work just as fine as before it was archived.
